python/ray/serve/tests/test_controller_recovery.py

```diff
@@ -365,11 +395,11 @@ def check_deleted():
     print("Finished killing the controller (with restart).")
 
     def check_controller_alive():
-        all_current_actors = list_actors(filters=[("state", "=", "ALIVE")])
-        for actor in all_current_actors:
-            if actor["class_name"] == "ServeController":
-                return True
-        return False
+        try:
+            ray.get(serve_instance._controller._get_logging_config.remote())
+            return True
+        except Exception:
+            return False
```

Comment:
the old method of checking if replica is alive  is more robust and direct, why change that?